unconvert.go

```diff
@@ -43,6 +44,7 @@ func apply(file string, edits editSet) {
 	}
 
 	fset := token.NewFileSet()
+	fmt.Printf("file: %q\n", file)
```

Comment:
Debug?

Reply:
Oops, I've missed it, thanks for catching this. Removed.